assets/style.css

```diff
@@ -0,0 +1,998 @@
+/* General definitions */
+
+@font-face {
+  font-family: "Roboto Condensed";
+  src: url(fonts/RobotoCondensed-Regular.ttf);
+}
+
+html * {
+  font-family: "Roboto Condensed", arial, sans-serif;
+}
+
+body {
+  background-color: #f4f5f6;
+  min-width: 450px;
+}
+
+.container,
+.container-fluid {
+  max-width: 1310px;
+}
+
+.bg-primary {
+  background-color: #f72210 !important;
+}
+
+.text-primary {
+  color: #f72210 !important;
+}
+
+.border *,
+.border {
+  border: solid 2px black !important;
+}
+
+.border-none {
+  border: none !important;
+}
```

Comment:
We can use the `border-*-0` bootstrap class here. Please check [here](https://hackerthemes.com/bootstrap-cheatsheet/#border-0) for more details

Reply:
Done! thanks for this one and the link for the cheatsheet ; )